ci/cloudbuild.yaml

```diff
@@ -53,14 +53,23 @@ steps:
   - 'TAGGED_VERSION=$TAG_NAME'
 
 options:
-  machineType: 'N1_HIGHCPU_32'
+  pool:
+    name: 'projects/solo-public/locations/us-central1/workerPools/envoy-gloo-runner'
 timeout: 20000s
 
 artifacts:
   objects:
     location: 'gs://solo-public-artifacts.solo.io/envoy/$COMMIT_SHA/'
     paths: ['linux/amd64/build_envoy_release/envoy']
 
+tags:
+  - "repo_envoy-gloo"
```

Comment:
Should we add a tag like `using-private-pool` so we can compare jobs run with/without private runners? I haven't spent much time using tags/filters on jobs in gcloud so I'm not sure whether that info is easily available

Reply:
This is accessible in the output of `gcloud builds describe <id>`. It is `{}` for a default pool run and non-empty for a private pool run. Additionally, it is a sort/filter option in the UI

Reply:
thanks! resolving the thread